From 9a2f831fd0427c1030a1656ab3d2d261eeb4870d Mon Sep 17 00:00:00 2001
From: Serhii Kupriienko
Date: Mon, 8 May 2023 13:39:38 +0300
Subject: [PATCH] Fix inspect atribute

---
 stone/frontend/ir_generator.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/stone/frontend/ir_generator.py b/stone/frontend/ir_generator.py
index d6db227..1107f52 100644
--- a/stone/frontend/ir_generator.py
+++ b/stone/frontend/ir_generator.py
@@ -1074,7 +1074,7 @@ class IRGenerator(object):
         assert issubclass(data_type_class, DataType), \
             'Expected stone.data_type.DataType, got %r' % data_type_class
 
-        argspec = inspect.getargspec(data_type_class.__init__)  # noqa: E501 # pylint: disable=deprecated-method,useless-suppression
+        argspec = inspect.getfullargspec(data_type_class.__init__)  # noqa: E501 # pylint: disable=deprecated-method,useless-suppression
         argspec.args.remove('self')
         num_args = len(argspec.args)
         # Unfortunately, argspec.defaults is None if there are no defaults
-- 
2.39.0

